src/serac/infrastructure/input.cpp

```diff
@@ -19,6 +19,7 @@ axom::inlet::Inlet initialize(axom::sidre::DataStore& datastore, const std::stri
 {
   // Initialize Inlet
   auto luareader = std::make_unique<axom::inlet::LuaReader>();
+  luareader->solState().open_libraries(sol::lib::math);
```

Comment:
https://github.com/LLNL/axom/blob/494fc6429ac217b9d6aff5c9bc967dd3ee8aa5b9/src/axom/inlet/LuaReader.hpp#L44-L47

Added 23 days ago to Axom and the TPLs were updated Dec 17th.  So if we rebuild the TPLs it will be there.  Do you want me to handle rebuilding them today? Easy to do during meetings...

Reply:
Eh, it's not a huge rush. Just making sure it went in.

Reply:
The new PR i just merged has the a LuaReader w/ this enabled so you can remove this line(s).